release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1678,10 +1678,30 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
 
- * Previously, some conditions during the startup process of the Cluster Autoscaler Operator caused a lock that prevented the Operator from successfully starting and marking itself available. As a result, the cluster became degraded. The issue is resolved with this release. (link:https://issues.redhat.com/browse/OCPBUGS-20038[*OCPBUGS-20038*])
+* Previously, some conditions during the startup process of the Cluster Autoscaler Operator caused a lock that prevented the Operator from successfully starting and marking itself available. As a result, the cluster became degraded. The issue is resolved with this release. (link:https://issues.redhat.com/browse/OCPBUGS-20038[*OCPBUGS-20038*])
 
 * Previously, the bootstrap credentials used to request client credentials for control plane nodes did not include the generic, all service accounts group. As a result, the cluster machine approver ignored certificate signing requests (CSRs) created during this phase. In certain conditions, this prevented approval of CSRs during bootstrap and caused the installation to fail. With this release, the bootstrap credential includes the groups that the cluster machine approver expects for a service account. This change allows the machine approver to take over from the bootstrap CSR approver earlier in the cluster lifecycle and should reduce bootstrap failures related to CSR approval. (link:https://issues.redhat.com/browse/OCPBUGS-8349[*OCPBUGS-8349*])
 
+* Previously, if scaling the machines on a Nutanix cluster exceeded the available memory to complete the operation, machines would get stuck in the `Provisioning` state and could not be scaled up or down. The issue is resolved in this release. (link:https://issues.redhat.com/browse/OCPBUGS-19731[*OCPBUGS-19731*])
+
+* Previously, for clusters on which the Control Plane Machine Set Operator is configured to use the `OnDelete` update strategy, cached information about machines caused the Operator to balance machines incorrectly and place them in an unexpected failure domain during reconciliation. With this release, the Operator refreshes this information immediately before creating new machines so that it correctly identifies the failure domains to place machines in. (link:https://issues.redhat.com/browse/OCPBUGS-15338[*OCPBUGS-15338*])
+
+* Previously, the Control Plane Machine Set Operator used the `Infrastructure` object specification to determine the platform type for the cluster. For clusters upgraded from {product-title} version 4.5 and earlier, this meant that the Operator could not correctly determine that a cluster was running on AWS, and therefore did not generate the `ControlPlaneMachineSet` custom resource (CR) as expected. With this release, the Operator uses the status platform type, which is populated on all clusters independent of when they were created, and is now able to generate the `ControlPlaneMachineSet` CR for all clusters. (link:https://issues.redhat.com/browse/OCPBUGS-11389[*OCPBUGS-11389*])
```

Comment:
```suggestion
* Previously, the Control Plane Machine Set Operator used the `Infrastructure` object specification to determine the platform type for the cluster. For clusters upgraded from {product-title} version 4.5 and earlier, this [what?... practice? situation?] meant that the Operator could not correctly determine that a cluster was running on AWS, and therefore did not generate the `ControlPlaneMachineSet` custom resource (CR) as expected. With this release, the Operator uses the status platform type, which is populated on all clusters independent of when they were created and is now able to generate the `ControlPlaneMachineSet` CR for all clusters. (link:https://issues.redhat.com/browse/OCPBUGS-11389[*OCPBUGS-11389*])
```
- Per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references), ensure that pronouns have clear references.
- Remove comma before subordinating conjunction `and`. A comma is only required before a coordinating conjunction.